test_util/launch_configs/aws-zen-cf.yaml

```diff
@@ -0,0 +1,19 @@
+---
+deployment_type: template
+deployment_name: This-is-a-test-stack
+template_url: https://s3.amazonaws.com/downloads.dcos.io/dcos/testing/master/cloudformation/coreos-zen-1.json
+provider: aws
+provider_info:
+    region: us-west-2
+    access_key_id: ALSAFLAJ4L20922KF20TL294
+    secret_access_key: ADWOFIAOFWJTPW3924JR
+key_helper: true
+zen_helper: true
+dcos_parameters:
+    AdminLocation: 0.0.0.0/0
+    PublicAgentInstanceCount: 2
+    PrivateAgentInstanceCount: 1
+    PublicAgentInstanceType: m4.xlarge
+ssh:
+    user: core
+    private_key_path: /tmp/aawpei3
```

Comment:
But if you call it `private_key` that implies that this could be the key contents themselves, right?

How about `private_key_file_path` ?